swift-sdk/IterableInboxViewController.swift

```diff
@@ -261,6 +280,21 @@ open class IterableInboxViewController: UITableViewController {
         }
         return DateFormatter.localizedString(from: createdAt, dateStyle: .medium, timeStyle: .short)
     }
+    
+    private func instantiateViewDelegate(withClassName className: String) {
+        guard let delegateClass = NSClassFromString(className) as? IterableInboxViewControllerViewDelegate.Type else {
+            // we can't use IterableLog here because this happens from storyboard before logging is initialized.
+            print("❤️: Could not initialize dynamic class: \(className), please check protocol \(IterableInboxViewControllerViewDelegate.self) conformanace.")
+            return
+        }
+        guard let delegateObject = delegateClass.createInstance?() else {
+            print("❤️: 'createInstance() method is not defined in \(className)'")
```

Comment:
Not a deal breaker for approval but are the single quotes here in the right place for this string?

Reply:
```suggestion
            print("❤️: 'createInstance()' method is not defined in \(className)'")
```